webpack.config.js

```diff
@@ -0,0 +1,42 @@
+
+  const path = require('path');
+  const HtmlWebpackPlugin = require('html-webpack-plugin');
+  const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+  module.exports = {
+   mode: 'development',
```

Comment:
Can we add a `watch: true` here as well?

Reply:
That is enabled by [default ](https://webpack.js.org/configuration/watch/#watch)